flake.nix

```diff
@@ -1,150 +1,45 @@
-        # Each block can be thought of as providing a "feature" to its cell.
-        # Cell blocks have types.
-        # Each cell block must be either:
-        #   A nix file named after the cell block
-        #   A directory named after the cell block and containing a default.nix
-        # Not all cells have the same cell blocks.
-        # All cell blocks belong in a cell.
-        #
-        # In this repository we have six cell blocks, listed below with their type:
-        #   devshells :: devshells
-        #     Development shells available via nix develop
-        #   packages :: installables
-        #     Derivations available via nix build
-        #   library :: functions
-        #     Everything that is not a derivation goes here
-        #     Includes functions, attrsets and simple literal values shared across cells
-        #     These are not exposed to the flake
-        #   scripts :: functions
-        #     Scripts that are available in the dev shell
-        #   tests :: functions
-        #     Scripts that are available in the dev shell
-        #   ciJobs :: installables
-        #     Jobsets for our Hydra CI
-        #
-        # std provides a TUI to interact with the cell blocks.
-        # Available interactions are determined by the cell block's type.
-        cellBlocks = [
-          (inputs.std.blockTypes.devshells "devshells")
-          (inputs.std.blockTypes.installables "packages")
-          (inputs.std.blockTypes.functions "library")
-          (inputs.std.blockTypes.installables "scripts")
-          (inputs.std.blockTypes.installables "tests")
-          (inputs.std.blockTypes.installables "ciJobs")
-          (inputs.std.blockTypes.runnables "operables")
-          (inputs.std.blockTypes.containers "oci-images")
-        ];
-      }
-
-      # The growOn function takes a variadic number of "soil" arguments, which
-      # provide a compatability layer for standard flake attributes. We
-      # translate (a.k.a. "harvest") the cell blocks into flake attributes.
-      # Successive arguments will be recursively merged.
-      {
-        # harvest the packages cell block from the marlowe-playground cell.
-        packages = inputs.std.harvest inputs.self [ "marlowe-playground" "packages" ];
-
-        # harvest the devshells cell block from the marlowe-playground cell.
-        devShells = inputs.std.harvest inputs.self [ "marlowe-playground" "devshells" ];
+    std.url = "github:divnix/std";
+    std.inputs.n2c.follows = "n2c";
+  };
 
-        # harvest the tests cell block from the marlowe-playground cell.
-        checks = inputs.std.harvest inputs.self [ "marlowe-playground" "tests" ];
 
-        # harvest the ciJobs cell block from the automation cell.
-        hydraJobs = inputs.std.harvest inputs.self [ "automation" "ciJobs" ];
+  outputs = inputs: inputs.iogx.lib.mkFlake {
+    inherit inputs;
+    repoRoot = ./.;
+    nixpkgsConfig = {
+      permittedInsecurePackages = [
```

Comment:
We should have comments explaining why these are needed and when they can be removed